resources/views/layouts/app.blade.php

```diff
@@ -25,7 +25,7 @@
     @stack('head')
     @config('design/head/includes')
 </head>
-<body class="text antialiased">
+<body class="text antialiased has-[.prevent-scroll:checked]:overflow-clip">
```

Comment:
What does this have to do with the country select?

Reply:
Not much, this PR has small frontend tweaks rather than focus on a country select

Reply:
Please separate it so we've a clean trackable changelog.